library/src/main/java/com/cleveroad/fanlayoutmanager/AnimationHelper.java

```diff
@@ -57,8 +57,11 @@ void shiftSideViews(@NonNull final Collection<ViewAnimationInfo> views,
     /**
      * Rotate view from custom radius to 0.
      *
-     * @param view     view to rotate
-     * @param listener animator listener to check start or end animation
+     * @param view            view to rotate
+     * @param customListener  custom animator listener to check start or end animation
+     * @param managerListener animator listener to notify layout manager of the animation state
      */
-    void straightenView(View view, @Nullable Animator.AnimatorListener listener);
+    void straightenView(View view,
+                        @Nullable Animator.AnimatorListener customListener,
+                        @Nullable Animator.AnimatorListener managerListener);
```

Comment:
Please explain why do you need a `managerListener` object here? If you need to use multiple listeners you can pass your own implementation of composite listener here (a listener that holds references to the list of another listeners and calls them once some event is triggered).